Session.test.js

```diff
@@ -0,0 +1,22 @@
+import { Session } from './Session';
+
+test('Check for correct value of sessionId', () => {
+    const sess = new Session(); 
+    sess.setSessionId("LYW23902"); 
+    const id = sess.getSessionId();
+    expect(id).toBe(sess.sessionId); 
+}); 
+
+test('Check for correct value of instanceIp', () => {
+    const sess = new Session();
+    sess.getIpOfVM("122.01.231.25");
```

Comment:
Im surprised this test actually passes. This doesnt make sense. Did you mean setIpOfVm?